loopy/transform/iname.py

```diff
@@ -1639,11 +1639,15 @@ def map_reduction(self, expr, expn_state):
 
 
 @for_each_kernel
-def make_reduction_inames_unique(kernel, inames=None, within=None):
+def make_reduction_inames_unique(kernel, inames=None, within=None,
+                                 do_not_propagate_metadata=False):
```

Comment:
Doesn't look like anything looks at this flag? (Do we need it? I feel like you'd almost always want tags to propagate. Or at least we could wait until a use case arises.)

Reply:
Yeah I agree, we almost always want to propagate the tags. Just got rid of the flag. Thanks!